### PR TITLE
Bug 1500724, updated Online Getting Started topics

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1500724

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@wgordon17 FYI

File: getting_started/basic_walkthrough.adoc [hunk 1]
```diff
@@ -122,14 +129,14 @@ image::gs-copy-webhook.png[Copy webhook]
 
 - In the {product-title}  web console:
 .. Navigate to the project containing your application.
-.. Click the *[ Browse ]* tab, then click *[ Builds ]*, then click the name of the
+.. Click the *Browse* tab, then click *Builds*, then click the name of the
 build for your Node.js application.
-.. From the *[ Configuration ]* tab, click image:copy.jpg[] next to *GitHub webhook
+.. From the *Configuration* tab, click image:copy.jpg[] next to *GitHub webhook
```
Comment (wgordon17) [inline getting_started/basic_walkthrough.adoc]: Please specify that the "Content Type" has to be "application/json", as opposed to the default "application/x-www-form-urlencoded"

screen shot 2017-10-17 at 15 28 52

File: getting_started/basic_walkthrough.adoc [hunk 2]
```diff
@@ -52,21 +54,26 @@ $ cd nodejs-ex
...
 [[bw-creating-a-new-application]]
 == Creating a New Application
 
 In this section, you will deploy your first application to {product-title} using
 the web console.
 
-. Navigate to the https://console.preview.openshift.com/console/[welcome
-page] of the {product-title} web console and click *[ New Project ]* to create
-your first project:
+. Navigate to the welcome page of the {product-title} web console and click *New
```
Comment (wgordon17) [inline getting_started/basic_walkthrough.adoc]: I believe this should "Create Project" as opposed to "New Project"

**Comment by ahardin-rh** [comment]
@wgordon17 Thanks! Updated.

**Comment by ahardin-rh** [comment]
No revision history necessary, as no revision histories are built in Online

**Pull request merged**